parsers/cobol.c

```diff
@@ -132,21 +227,37 @@ static void initializeCobolParser (langType language)
 	cobol = language;
 
 	addLanguageCallbackRegex (cobol,
-					  COBOL_REGEX_PREFIX
-					  "[0-9]+[ \t]+([A-Z0-9][A-Z0-9-]*)[ \t]+("
-					  "BLANK|OCCURS|IS|JUST|PIC|REDEFINES|RENAMES|SIGN|SYNC|USAGE|VALUE"
-					  ")",
-					  "{icase}",
-					  make_tag_for_data_maybe, NULL, &cobol);
+		COBOL_REGEX_PREFIX_FIXED
```

Comment:
Can we use COBOL_REGEX_PREFIX_UNIFIED here.
So you can uinfy two addLanguageCallbackRegex into one.